shared/templates/service_enabled/tests/service_disabled.fail.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# packages = {{{ PACKAGENAME }}}
+
+systemctl stop {{{ DAEMONNAME }}}
+systemctl disable {{{ DAEMONNAME }}}
+{{% if MASK_SERVICE %}}
```

Comment:
Unlike the  `service_disabled` template, the `service_enabled`  template doesn't use the `MASK_SERVICE` parameter.